frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -99,14 +99,20 @@ export const ClusterServiceVersionTableHeader = () => {
     },
     {
       title: 'Namespace',
+      sortField: 'metadata.namespace',
+      transforms: [sortable],
       props: { className: tableColumnClasses[1] },
     },
     {
       title: 'Status',
+      sortField: 'status.phase',
```

Comment:
We probably need a `sortFunc` that returns the same values as the `ClusterServiceVersionStatus`. Otherwise the sort won't always match the display value.

Reply:
Good catch.  It occurs to me we're also combining `ClusterServiceVersionTableRow` and `SubscriptionTableRow` in the table, which is another mismatch since the latter outputs a warning or `subscription.status.state` in the status column.  I'm not sure how to resolve.

Reply:
Given the complexity of the combined rows, I'm going to punt on making `status` sortable at this time.  There are additional changes coming to OLM that may remove subscription rows anyhow.